parse.go

```diff
@@ -370,21 +375,23 @@ func (p *Parser) Parse(args []string) error {
 	}
 	var subParser *Parser
 	if len(args) == 0 {
+		p.Invoked = true // when there is any match, it's invoked, or the default action will be called
```

Comment:
this is where we got different, but I don't think this is the right place for `Invoked` to be